### DISCUSSION TITLE
How to change the Icons package

### DISCUSSION THREAD

**Discussion by GVALFER** [zed-industries/zed#13256] [Feature Requests]
Hi,

Im new using Zed Editor. 
I want change the icons package, how can i do that?

 Cheers

**Comment by JosephTLyons** [comment]
This isn't currently possible, without forking your own copy of Zed and modifying it.

**Comment by smir-ant** [comment]
up! 🙏

i very need and want it (and i think not only me)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
